### DISCUSSION TITLE
Template variable "baseDir" is empty

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [renovatebot/renovate#8581] [Request Help] [answered]
To include `baseDir` in the pull request title and branch name, I wrote the following renovate.json

[renovate.json](https://github.com/suzuki-shunsuke/test-renovate/blob/61ba05cd83ff28a9c3c14100bf7b4fddb4dc24ea/renovate.json)

```json
{
  "packageRules": [
    {
      "matchPaths": ["bar/*"],
      "additionalBranchPrefix": "{{baseDir}}-",
      "commitMessageSuffix": "({{baseDir}})"
    }
  ]
}
```

bar/Dockerfile

```dockerfile
FROM golang:1.15.0
```

Then the following pull request was created.

https://github.com/suzuki-shunsuke/test-renovate/pull/85

The pull request title is `Update golang Docker tag to v1.15.8 ()`.

I expect the suffix of the pull request title is `(bar)`, but actually `()`.
On the other hand, the pull request branch name is `renovate/bar-golang-1.x`, which is expected.

Is the Renovate setting wrong?

---

ref. https://docs.renovatebot.com/templates/

> baseDir: The full directory with path that the dependency has been found in

**Comment by suzuki-shunsuke** [comment, marked as answer]
https://github.com/renovatebot/renovate/pull/8625

`baseDir` was removed from the document. https://docs.renovatebot.com/templates/
We should use `packageFileDir` instead of `baseDir`, right?

**Reply by rarkins** [reply]
Yes, but any baseDir templates you have should get automatically migrated
